.circle/data/ds005_outputs.txt

```diff
@@ -45,6 +45,23 @@ ds005/out/fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-MN
 ds005/out/fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-MNI152NLin2009cAsym_preproc.nii.gz
 ds005/out/fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-MNI152NLin2009cAsym_variant-smoothAROMAnonaggr_preproc.nii.gz
 ds005/out/fmriprep/sub-01.html
+ds005/out/fmriprep/sub-01/figures
+ds005/out/fmriprep/sub-01/figures/_scratch_reportlets_fmriprep_sub-01_anat_sub-01_T1w_reconall.svg
```

Comment:
Can we drop the `_scratch_reportlets_fmriprep_`?

Also, if this is going to be a derivative, I think it makes sense to apply some naming conventions that can eventually be standardized. For instance, perhaps the type can be `_{static,anim}.svg` indicating whether it's a static or animated image. Here are some examples, but feel free to reject any that you don't like.

Structural:

* reconall.svg -> surfaceoutline_static.svg
* t1_2_mni.svg -> normalization_anim.svg
* t1_seg.svg -> segmentation_static.svg
* t1_skull_strip.svg -> skullstrip_static.svg

Functional:

* acompcor.svg -> acompcor_static.svg
* bbr.svg -> registration_anim.svg
* bold_mask.svg -> skullstrip_static.svg
* syn_sdc.svg -> synsdc_anim.svg
* tcompcor.svg -> tcompcor_static.svg

Reply:
I totally agree with all of this. Would it be fine with you if we leave this comment for the next iteration over #478 ? Just to make all these changes more iteratively, and risk less to breaking reports.

Reply:
Missed this, sorry. Sure.